recipes/dust_extinction/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "dust_extinction" %}
+{% set version = "0.7" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "68c096e60b133cfaa60e641cd6a1c9b45a0f9b82d2cd4c8560ce68db3182373d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 1000
```

The build number should start from `0` for new versions.
```suggestion
  number: 0
```